audio/analyzer.py

```diff
@@ -111,21 +112,67 @@ def analyze_album(album_path, upload_spectrogram=False):
     flac_files = [file for file in os.listdir(album_path) if file.lower().endswith('.flac')]
     album = {}
 
+    # Ask the user for input if upload_spectrogram is True
+    tracks_to_upload_spectrograms = set()
+    if upload_spectrogram:
+        print("Enter the track numbers to upload spectrals separated by commas, or '*' for all tracks:")
+        user_input = input().strip()
+        if user_input == '*':
```

Comment:
Instead of checking if the input is exactly `*` id check if it is a `int` so the default behaviour is just to upload all and it doesn't shit itself if someone enters a letter 

```py
 if !isinstance(user_input, int):
            # Use a wildcard to indicate all tracks for spectrogram upload
            tracks_to_upload_spectrograms = {'*'}
        else:
            # Parses user input, keeping the disc prefix and removing leading zeros from track numbers
```